Tests/varLib/iup_test.py

```diff
@@ -0,0 +1,63 @@
+import fontTools.varLib.iup as iup
+import sys
+import pytest
+
+test_cases = [
+
+
+]
+
+class IupTest:
+
+    def setUp(self):
+        pass
+
+    def tearDown(self):
```

Comment:
You don't need these two no-op setUp/tearDown methods, pytest discover the test class by its name ending in "Test"